src/wp-admin/includes/template.php

```diff
@@ -2009,7 +2013,12 @@ function settings_errors( $setting = '', $sanitize = false, $hide_on_update = fa
 			esc_attr( $details['type'] )
 		);
 
-		$output .= "<div id='$css_id' class='$css_class'> \n";
+		$aria_live_attr = '';
+		if ( ! empty( $details['aria_live'] ) && in_array( $details['aria_live'], array( 'off', 'polite', 'assertive' ), true ) ) {
```

Comment:
This can be simplified:

```suggestion
		if ( in_array( $details['aria_live'], array( 'off', 'polite', 'assertive' ), true ) ) {
```